catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -214,6 +216,11 @@ private Metacard transformPdf(String id, PDDocument pdfDocument, String contentI
         extractPdfMetadata(pdfDocument, metacard);
 
         metacard.setThumbnail(generatePdfThumbnail(pdfDocument));
+
+        String wktFromPDF = GEO_PDF_PARSER.getWktFromPDF(pdfDocument);
```

Comment:
How about this? 

Optional.ofNullable(wktFromPdf).ifPresent(metacard::setLocation);